src/index.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{.Locale}}">
```

Comment:
@kosssi this is not the PR you are looking for --> https://github.com/cozy/cozy-files-v3/pull/56

![](https://cdn.meme.am/instances/500x/65618649.jpg)